builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -15,14 +15,22 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             throws IOException, ServletException {
         HttpServletRequest request = (HttpServletRequest) req;
         String auth = request.getHeader("Authorization");
-        if(auth == null ){
+        String uri = request.getRequestURI();
+
+        if (uri.endsWith("/api/system/state")) {
+            System.out.println("Che server status endpoint should not be secured: " + uri);
```

Comment:
I agree that we should be using the standard logging framework, but this isn't necessarily the place to change it.

As it stands I feel like the filter logs too much anyways, many of these messages should be moved to the `debug` level.